src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+
```

Comment:
Don't leave an empty line here for consistency since all other classes don't leave an empty line after the opening bracket